SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -110,6 +110,8 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	FDelegateHandle OnPropertyChangedDelegateHandle;
 	bool bStopSpatialOnExit;
 
+	bool bSchemaBuildError;
```

Comment:
Is this flag persisted across editor sessions ?

Reply:
No - it was decided that because those files are tracked by source control it might be difficult to know whether a schema is actually valid or not!